src/libs/export.c

```diff
@@ -1303,6 +1303,32 @@ void gui_init(dt_lib_module_t *self)
   dt_bauhaus_combobox_add(d->intent, _("absolute colorimetric"));
   gtk_box_pack_start(GTK_BOX(self->widget), d->intent, FALSE, TRUE, 0);
 
+  tooltip = g_strdup_printf(_("• perceptual : "
+                              "smoothly moves out-of-gamut colors into gamut,"
```

Comment:
perceptual is often a proprietary LUT that depends on the destination space. Usually, it compresses the whole gamut range such that the color transitions are preserved indeed.

Reply:
Ok.